R/volcano_protti.R

```diff
@@ -73,14 +75,21 @@ volcano_protti <- function(data, grouping, foldchange, significance, method, pro
         y = "-log10(p-value)"
       ) +
       geom_hline(yintercept = -1 * log10(significance_cutoff), linetype = "dashed") +
-      geom_vline(xintercept = foldchange_cutoff, linetype = "dashed") +
-      geom_vline(xintercept = -1 * foldchange_cutoff, linetype = "dashed") +
+      geom_vline(xintercept = log2(foldchange_cutoff), linetype = "dashed") +
+      geom_vline(xintercept = -1 * log2(foldchange_cutoff), linetype = "dashed") +
       theme(legend.position = "none")  +
       theme_bw() +
       scale_x_continuous(breaks = seq(round(-1 * max(abs(dplyr::pull(data, {{foldchange}})), na.rm = TRUE) - 0.5, 0), round(max(abs(dplyr::pull(data, {{foldchange}})), na.rm = TRUE) + 0.5, 0), 1)) +
       coord_cartesian(xlim = c(round(-1 * max(abs(dplyr::pull(data, {{foldchange}})), na.rm = TRUE) - 0.5, 0), round(max(abs(dplyr::pull(data, {{foldchange}})), na.rm = TRUE) + 0.5, 0)))
+  if (interactive == TRUE)
+    {
     return(plotly::ggplotly(plot))
```

Comment:
actually I think we should not call the plot just "plot" this caused some problems in the past for me since there is a function called plot. Don't know if it is problematic here, but I think we should be safe by calling it plot_volcano or so..